src/tmx/TmxUtils/src/database/DbConnectionPool.cpp

```diff
@@ -120,17 +120,23 @@ void DbConnectionPool::SetConnectionUrl(std::string connectionUrl) {
 }
 
 std::string DbConnectionPool::GetPwd(){
-	const char* EnvVar = "MYSQL_ROOT_PASSWORD";
+	const char* EnvVar = "MYSQL_PASSWORD";
```

Comment:
This is an improvement over what we have currently.  Are we going to revisit the strategy of using environment variables for passwords when we implement security because IMO this is still a risk because anybody that gains access to the operating system can query for it?

Reply:
Slight change is that now the environment variable points to a file which is mounted to the container. The actual file can be located on an HSM, the environment variable is only necessary to pass the file location to the container.